### ISSUE TITLE
Framework test suite fails on clean install with new bootstrap

### ISSUE THREAD

**Issue by rjeschmi** [hpcugent/easybuild-framework#1670]
Testing a clean install with the bootstrap has a couple failures

```
======================================================================
ERROR: Test fix_broken_easyconfigs.py script.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/test/fram
ework/scripts.py", line 172, in test_fix_broken_easyconfig
    run_cmd(script_cmd)
  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuild
/tools/run.py", line 149, in run_cmd
    return parse_cmd_output(cmd, stdouterr, ec, simple, log_all, log_ok, regexp)
  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuild
/tools/run.py", line 397, in parse_cmd_output
    raise EasyBuildError('cmd "%s" exited with exitcode %s and output:\n%s', cmd, ec, stdouterr)
EasyBuildError: 'cmd "PYTHONPATH=/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-
2.6.0-py2.6.egg:$PYTHONPATH:/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0
-py2.6.egg/test/framework/sandbox /tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework
-2.6.0-py2.6.egg/easybuild/scripts/fix_broken_easyconfigs.py /tmp/eb-if9nvq/eb-SEtRiK/eb-EII4qf/eb-8ARStg/eb-zBm5y8/eb
-rOTRc4/eb-dsI8Yz/eb-w839Uk/nosuchsoftware.eb" exited with exitcode 1 and output:\nTraceback (most recent call last):\
n  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuil
d/scripts/fix_broken_easyconfigs.py", line 33, in <module>\n    from vsc.utils import fancylogger\n  File "/tmp/ebtest
/software/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.18-py2.6.egg/vsc/__init__.py", line 29, in <module>
\n    import pkg_resources\nImportError: No module named pkg_resources\n'

======================================================================
ERROR: Test for generate_software_list.py script.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/test/fram
ework/scripts.py", line 82, in test_generate_software_list
    out, ec = run_cmd(cmd, simple=False)
  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuild
/tools/run.py", line 149, in run_cmd
    return parse_cmd_output(cmd, stdouterr, ec, simple, log_all, log_ok, regexp)
  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2.6.egg/easybuild
/tools/run.py", line 397, in parse_cmd_output
    raise EasyBuildError('cmd "%s" exited with exitcode %s and output:\n%s', cmd, ec, stdouterr)
EasyBuildError: 'cmd "/usr/bin/python /tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_frame
work-2.6.0-py2.6.egg/easybuild/scripts/generate_software_list.py --local --quiet --path /tmp/eb-if9nvq/eb-SEtRiK/eb-EI
I4qf/eb-8ARStg/eb-zBm5y8/eb-rOTRc4/eb-dsI8Yz/eb-Me3iQn/eb-j3x_WR/tmpLKcQlu" exited with exitcode 1 and output:\nTraceb
ack (most recent call last):\n  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_frame
work-2.6.0-py2.6.egg/easybuild/scripts/generate_software_list.py", line 36, in <module>\n    import easybuild.tools.co
nfig as config\n  File "/tmp/ebtest/software/EasyBuild/2.6.0/lib/python2.6/site-packages/easybuild_framework-2.6.0-py2
.6.egg/easybuild/tools/config.py", line 43, in <module>\n    from vsc.utils import fancylogger\n  File "/tmp/ebtest/so
ftware/EasyBuild/2.6.0/lib/python2.6/site-packages/vsc_base-2.4.18-py2.6.egg/vsc/__init__.py", line 29, in <module>\n 
   import pkg_resources\nImportError: No module named pkg_resources\n'

----------------------------------------------------------------------
Ran 383 tests in 1944.928s
```

**Comment by rjeschmi** [comment]
@boegel tracked it down to some fiddling with PYTHONPATH in the tests

https://github.com/hpcugent/easybuild-framework/blob/master/test/framework/scripts.py#L53

**Comment by boegel** [comment]
Thanks for reporting this @rjeschmi.

I was able to reproduce this on a 'bare' CentOS6 (where `setuptools` is not available on the system).

Fix is pretty straightforward, see #1671; let's follow up there
